src/GrafanaDashboardDeploy/tests/resources/folder 10 xxmzSkJZk.json

```diff
@@ -0,0 +1,15 @@
+﻿{
+    "id": 10,
+    "uid": "xxmzSkJZk",
```

Comment:
>xxmzSkJZk [](start = 12, length = 9)

Can we pick whatever UID we want? Can we call it "engstage" or something?

Reply:
Looks like yes, it can be anything that matches [`^[a-zA-Z0-9\-\_]*$`](https://github.com/grafana/grafana/blob/416562bd9062fbf5cf142b717a52721dbc47b86c/pkg/util/shortid_generator.go#L11). I'll add something to the documentation about choosing UIDs.